_targets_eval_postprocessing.R

```diff
@@ -1144,6 +1226,92 @@ scenario_targets <- list(
   )
 )
 
+# Real-time relative targets--------------------------------------
+real_time_rel_targets <- list(
+  tar_target(
+    name = real_time_crps_both_models,
+    command = score_real_time_outputs(
+      score_type = "crps",
+      real_time_output_dir = eval_config$real_time_output_dir,
+      table_of_run_ids = as.data.frame(eval_config$table_of_run_ids),
+      locations = unique(eval_config$location_ww),
+      dates = as.character(seq(
+        from = lubridate::ymd("2024-02-05"),
+        to = lubridate::ymd("2024-03-11"),
+        by = "week"
+      )),
+      eval_data = eval_hosp_data
+    )
+  ),
+  tar_target(
+    name = real_time_wis_hosp_only,
+    command = score_real_time_outputs(
+      score_type = "wis",
+      real_time_output_dir = eval_config$real_time_output_dir,
+      table_of_run_ids = as.data.frame(eval_config$table_of_run_ids),
+      locations = unique(eval_config$location_ww),
+      dates = as.character(seq(
+        from = lubridate::ymd("2024-02-05"),
+        to = lubridate::ymd("2024-03-11"),
+        by = "week"
+      )),
+      eval_data = eval_hosp_data,
+      hosp_only = TRUE
+    )
+  ),
+  tar_target(
+    name = real_time_wis_both_models_raw,
+    command = combine_hub_and_local_wis(
+      cfa_real_time_scores,
+      real_time_wis_hosp_only
+    )
+  ),
+  # For the real-time comparison, we exclude the forecasts that are the
+  # same
+  tar_target(
+    name = real_time_wis_both_models,
+    command = real_time_wis_both_models_raw |>
+      dplyr::anti_join(ww_forecast_date_locs_to_excl) |>
+      # Could eventually replace this with what is on the Hub
+      dplyr::left_join(table_of_loc_dates_w_ww) |>
+      dplyr::filter(ww_sufficient)
+  ),
+  tar_target(
+    name = rel_real_time_wis,
+    command = get_rel_wis_real_time(real_time_wis_both_models)
+  ),
+  tar_target(
+    name = overall_real_time_rel_wis,
+    command = rel_real_time_wis |>
+      dplyr::ungroup() |>
+      dplyr::summarise(mean_rel_wis = mean(rel_wis, na.rm = TRUE))
```

Comment:
we are not presenting this in the text anymore, can remove. I previously was keeping track of both since we had both in the different places in the text